mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -465,6 +465,36 @@ private[python] class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib LDA.run()
+   */
+  def trainLDAModel(
+      data: JavaRDD[LabeledPoint],
```

Comment:
Umm, it is a little difficult to decide which is the better. The different point between yours and mine from the users point of view are :

- Each row type is an array or a tuple
- Feature data type is array or DenceVector/SparceVector
- Does yours support sparce vector?